includes/class-freemius.php

```diff
@@ -5484,7 +5488,11 @@ private function parse_settings( &$plugin_info ) {
                 'affiliate_moderation' => $this->get_option( $plugin_info, 'has_affiliation' ),
                 'bundle_id'            => $this->get_option( $plugin_info, 'bundle_id', null ),
                 'bundle_public_key'    => $this->get_option( $plugin_info, 'bundle_public_key', null ),
-                'opt_in_moderation'    => $this->get_option( $plugin_info, 'opt_in', null ),
+                'opt_in_moderation'    => $this->get_option(
+                    $plugin_info,
+                    'opt_in',
+                    $this->get_option( $plugin_info, 'opt_in_moderation', null )

```

Comment:
Perhaps just add a comment here that we support both parameter names `opt_in` and `opt_in_moderation` for backward compatibility.